libraries/AP_Camera/examples/ReadMe.md

```diff
@@ -0,0 +1,79 @@
+
+# Real-Time Object Tracking with Gimbal Control (Simulation)
+
+## Overview
+
+## Requirements and Installations
+1. We assume you have already setup Ardupilot with SITL and gazebo simulation
+2. Install Gstreamer
+
+```
+sudo apt update
+```
+
+Install build tools and Python development libraries and GStreamer related plugins
+```
+
+sudo apt install -y build-essential cmake git pkg-config libgtk-3-dev libavcodec-dev libavformat-dev libswscale-dev
+
+sudo apt install -y python3-dev python3-numpy python3-pip
+
+sudo apt install -y libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev \
+                    gstreamer1.0-plugins-good gstreamer1.0-plugins-bad \
+                    gstreamer1.0-plugins-ugly gstreamer1.0-libav gstreamer1.0-tools
+```
+
+3. default opencv from pip should be uninstalled and again built with gstreamer support (If already did this go to next step)
```

Comment:
I think we should avoid requiring a custom build of opencv to include gstreamer support, and instead use standard packages. We can add a class to convert from the gstreamer frames to ones compatible with opencv, such as the example here: https://www.ardusub.com/developers/opencv.html